.github/workflows/build-ios.yml

```diff
@@ -0,0 +1,186 @@
+name: Build IOS
+on:
+  workflow_call:
+    inputs:
+      version_file:
+        type: string
+        required: true
+      macos_version:
+        type: string
+        required: true
+      xcode_version:
+        type: string
+        required: true
+      version:
+        type: string
+        required: true
+      prefix:
+        type: string
+        required: true
+env:
+  GOPRIVATE: github.com/getlantern
+  S3_BUCKET: lantern
+jobs:
+  build-ios:
+    runs-on: ${{ inputs.macos_version }}
+    permissions:
+      contents: "read"
+      id-token: "write"
+    env:
+      version: ${{ inputs.version }}
+      version_file: ${{ inputs.version_file }}
+      prefix: ${{ inputs.prefix }}
+      AC_USERNAME: accounts@getlantern.org
+      AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+      - name: Pull LFS objects
+        run: git lfs pull
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: "go.mod"
+
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: ${{ inputs.xcode_version }}
+
+      - name: Repo access
+        run: |
+          mkdir /tmp/cache
+          echo "machine github.com login ${{ secrets.GH_TOKEN }} password x-oauth-basic" > /tmp/cache/.netrc
+          chmod 600 /tmp/cache/.netrc
+
+      - name: Setup Sentry CLI
+        uses: mathieu-bour/setup-sentry-cli@v1
```

Comment:
I think there's a `setup-sentry-cli@v2` available if you wanna upgrade it while you're in the area

Reply:
Done, Also upgraded other packages to latest.